lib/util/pssh.js

```diff
@@ -31,8 +31,9 @@ goog.require('shaka.util.Uint8ArrayUtils');
 shaka.util.Pssh = class {
   /**
    * @param {!Uint8Array} psshBox
+   * @param {boolean=} extractFromMoov boolean to extract pssh from moov atom
```

Comment:
Based on my reading of Mp4Parser, I think you can do without this flag.

Reply:
Yes, I'm going to remove it.